plugins/catalog-react/src/alpha/blueprints/EntityContentBlueprint.ts

```diff
@@ -40,30 +42,35 @@ export const EntityContentBlueprint = createExtensionBlueprint({
     coreExtensionData.routeRef.optional(),
     entityFilterFunctionDataRef.optional(),
     entityFilterExpressionDataRef.optional(),
+    entityContentGroupDataRef.optional(),
   ],
   dataRefs: {
     title: entityContentTitleDataRef,
     filterFunction: entityFilterFunctionDataRef,
     filterExpression: entityFilterExpressionDataRef,
+    group: entityContentGroupDataRef,
   },
   config: {
     schema: {
       path: z => z.string().optional(),
       title: z => z.string().optional(),
       filter: z => z.string().optional(),
+      group: z => z.literal(false).or(z.string()).optional(),
     },
   },
   *factory(
     {
       loader,
       defaultPath,
       defaultTitle,
+      defaultGroup,
       filter,
       routeRef,
     }: {
       loader: () => Promise<JSX.Element>;
       defaultPath: string;
       defaultTitle: string;
+      defaultGroup?: keyof typeof defaultEntityContentGroups;
```

Comment:
Noting for followup: I think we should allow `| string` here too, to allow for organic creation of new groups as well as custom groups internally

Reply:
Followup in https://github.com/backstage/backstage/pull/28916